services/utils/socrata.py

```diff
@@ -1,5 +1,7 @@
+from datetime import datetime
 import os
 
+import arrow
```

Comment:
> Arrow is named after the arrow of time and is heavily inspired by moment.js and requests.

Inspired by moment is a +1 in my book. This is a new library to me - TIL.

Reply:
👍 It's not the most performant date/time library out there, but the API is really nice.